src/pages/Human_Practices.pug

```diff
@@ -134,6 +134,40 @@ block article
+        Dr. Martin Zimmermann is not only the study coordinator of the Biological Sciences faculty, but he is also an expert in many biological fields including biological analysis methods.
+        During the whole project, we repeatedly exchanged E-Mails, where he answered our questions with much detail and knowledge.
+        We met with Dr. Zimmermann at the end of Mai. He introduced denaturated Polyacrylamide gels to us and helped us establish the right analytics for our samples. Among other things, we discussed gel runtime, gel size, the right voltage for our samples, and staining methods of our gels.
+        To get a better resolution Dr. Zimmermann advised us to use TBE buffer, which has a higher buffer capacity.
+        Furthermore, he suggested silver staining as a sustainable method to stain PAA gels and longer primers to establish better reaction conditions for our Enzym, which we implemented rapidly.
+        At the end of our Meeting, Dr. Zimmermann once more drew attention to Polyacrylamide. He advised us to be extra careful when handling the neurotoxic chemical. Click here to see our safety guidelines.
+
+    :markdown-it(html)
+    
+        # Meeting with Dr. Sebastian Palluk
+    
+        Dr. Palluk is co-founder and CTO of Ansa Biotechnologies, a startup developing enzymatic DNA synthesis technologies. When we were younger he participated in the iGEM competition himself and found it challenging that synthesizing DNA was such a difficult task. Nowadays he works with a genetically modified TdT for biological Applications and aims to improve the synthesis system for future implications.
+        Mr. Palluk was probably one of our most important experts. We met with him three times in total and had intensive E-Mail contact during the whole project.
+        In our first meeting, we started off presenting our ideas and two approaches to encode data on DNA.
+        Furthermore, we discussed four different ideas to prevent the secondary structure formation of our ssDNA primers and the affinity of the TdT for 3´OH base modifications.
+        Mr. Palluk commented on our ideas on immobilization and explained briefly streptavidin immobilization on magnetic beads as well as sequencing techniques like Nanopore sequencing and capillary-electrophorese to view the length distribution.
+        To conclude our first meeting, Mr. Palluk gave us a great overview of many important tasks we have to tackle in the future and was sure that we had all the resources to establish a unique interdisciplinary and round project, but that would probably be much harder to implement as we thought.
+
+        In our second meeting with Mr. Palluk, at the beginning of May, we presented our possibilities to hinder secondary structures.
+        He recommended starting our first experiments in solution with purification steps to establish our methods. Furthermore, he emphasized parallelizing the immobilization while still doing our first experiments.
```

Comment:
```suggestion
        He recommended starting our first experiments in solution with purification steps to establish our methods.
        Furthermore, he emphasized that it would be efficient to start working on the immobilization while conducting our first experiments.
```